bindata/assets/alerts/podsecurity-violations.yaml

```diff
@@ -18,7 +18,23 @@ spec:
           Refer to Kubernetes documentation on Pod Security Admission to learn more about these
           violations.
       expr: |
-        sum(increase(pod_security_evaluations_total{decision="deny",mode="audit",resource="pod"}[1d])) by (policy_level) > 0
+        sum(increase(pod_security_evaluations_total{decision="deny",mode="audit",resource="pod",ocp_namespace="non-platform"}[1d])) by (policy_level, ocp_namespace) > 0
       labels:
         namespace: openshift-kube-apiserver
         severity: info
+    - alert: PodSecurityViolation
+      annotations:
+        summary: One or more workloads users created in the cluster don't match their Pod Security profile
+        description: >-
```

Comment:
What would be the course of action if this alert fires? Contacting the support?

Reply:
Identify the workload by the procedure documented in https://docs.openshift.com/container-platform/4.12/authentication/understanding-and-managing-pod-security-admission.html#security-context-constraints-psa-alert-eval_understanding-and-managing-pod-security-admission and contact the developers of the failing workload.

Reply:
Even for platform namespaces?
It would be good to add a runbook to https://github.com/openshift/runbooks/tree/master/alerts/cluster-kube-apiserver-operator + a `runbook` _url annotation to the alert.

Reply:
Even for platform namespaces, the developers of the failing workloads are us in that case.

I had no idea about the runbooks, I'll see what we can do there :eyes: